### DISCUSSION TITLE
Onboarding new team members

### DISCUSSION THREAD

**Discussion by desputinski** [tcfev/forDem#52] [General]
Hi!

*Make sure you are protecting your identity before posting here. (make a new account on github)

Please introduce yourself in this thread, your skills and your interests and check our projects for a place where you might fit best: https://github.com/orgs/tcfev/repositories

We need help in backend, frontend, design, strategy, architecture, management and public engagement.
*You can also define your own field, convince us, and start your work there. 

Becoming a member of our parent organisation is easy (but you don't have to) 
- [Membership](https://forms.gle/3kRpb64fsk3ynEJp8)

Becoming one of us over Github is easier. Just let us know here in this thread that you are in and let's make a plan of action. 

Contributing to our projects is super easy. Just work on one of the issues or make a new one, clone/fork, add things, commit and make PR. 

Want to chat in private or have a video call?
- send us an email over: Fordem@transcf.org
- or simply post to this discussion. 
 
Let's see what we can do together. We are excited!

**Comment by chandansgowda** [comment]
Hello @desputinski 
I recently came across this project. May I know more about it ?

**Reply by armantorkzaban** [reply]
Hi @chandansgowda, welcome. 
forDem is a censorship-defiant sociopolitical platform being made for sociopolitical mobilisation. The aim of the platform is to promote democratic states and democratisation of states, next to education of the population. 
The platform is a project of TCF e.V., a civics and humans rights innovation lab, and an international team with different expertises is behind the effort. 
Let us know what you think about the project, how you found us, and what makes you interested.

**Reply by chandansgowda** [reply]
Hey @armantorkzaban 
Thanks for a great intro. I was looking out for open source projects to contribute and I came across forDem. The idea itself is great and I'll be happy to contribute to the project.

**Reply by armantorkzaban** [reply]
Hi @chandansgowda, thanks for your reply. This sounds really good.
I would suggest that you begin with github.com/tcfev and read a bit about the organisation and then move on to our issues (and feature list) here: github.com/tcfev/fordem
Later on you can find where you feel fit best and start with an issue. 
Our communication channels are issues over GH and Session app. 
In case you are interested to start, send your Session key via email of the org. and let's begin.

**Reply by desputinski** [reply]
@chandansgowda Hi! Great, if we can have you on board! If you're interested we can phone one of these days and I or Arman can explain in more detail about the project.

**Reply by chandansgowda** [reply]
Hey @desputinski @armantorkzaban,
I'm interested to work on the project. I'm actually busy till Wednesday. It would be great if the phone call is scheduled anyday after Wed.
Thanks

**Reply by armantorkzaban** [reply]
Hi @chandansgowda, have you received our email?

**Reply by chandansgowda** [reply]
Hey @armantorkzaban 
I was actually busy since we last spoke. I have replied to your mail.
Thanks

**Reply by armantorkzaban** [reply]
Hey @chandansgowda, you missed our appointment for today.

**Reply by chandansgowda** [reply]
Hey @armantorkzaban 
I got confused about the timings and thought it was at 8PM IST. Are you still available?

**Comment by desputinski** [comment]
https://www.worldtimebuddy.com/ist-to-cet-converter

So when it is 7 p.m. IST, it is 2:30 p.m. MEZ (Arman, Christopher)

**Comment by IranNewsWatch** [comment]
Hello everyone. I'm glad to join!

**Reply by desputinski** [reply]
Hello :)

**Reply by armantorkzaban** [reply]
Hi there. Welcome!
Please let others know about your project. 
Take a look around and see our projects and their issues. (some are hidden, we will show you those once you're finished with the public ones. ;-))

**Reply by IranNewsWatch** [reply]
Sorry I missed the notifications. I have created a simple automation tool which automates Iran protest news propagation (and filtering) here https://twitter.com/IranNewsWatch
Main repo here: https://github.com/IranNewsWatch/irannewsbot

I code in python, SQL. I understand basic HTML and hot to build dashboards using dash-plotly. Interested in twitter stuff, Knowledge management/categorization, e-demogracy

**Reply by armantorkzaban** [reply]
I love your project. 
What do you think about a robot that works like this: 
1. called with @forDemRobo+[tag/topic] -> it imports the post above it to a list for a moderator to import in forDem under [tag/topic]
2. user connects it with their account and it crawls and imports all the user's tweets in a temporary list, waiting for confirmation/categorisation of the user to publish on forDem

**Reply by IranNewsWatch** [reply]
on number 1:
The form of the UI matters. However, the import module in the backend is going to be pretty much the same. I can work on that part, i.e.: 1. The bot gets triggered by a mention. 2. Imports the tweet (and mentions) with the link to the tweet. 
In term of planning, let's make sure moderators will eventually get replaced by our AI concept categorization, etc

**Reply by armantorkzaban** [reply]
Shall we begin? @IranNewsWatch

**Comment by osAIran** [comment]
Hello there 
I'd be glad to join you as well.
You can find me at we.are.jina.amini@proton.me

**Reply by armantorkzaban** [reply]
Welcome!

**Reply by armantorkzaban** [reply]
How/where would you like to begin?

**Reply by desputinski** [reply]
We could also brief you on what what parts of the project you can take part, where it makes sense. Maybe chat or phone... We have a chat group on the secure messenger session for example (https://getsession.org/)

**Reply by osAIran** [reply]
I think its best for me to begin with the discussion I made about voting and the coalition platform. 
That sounds good, lets have a quick chat

**Reply by IranNewsWatch** [reply]
Hi Jina. Welcome.  Where can I find the "discussion ... about voting and the coalition platform"?

**Reply by osAIran** [reply]
https://github.com/tcfev/forDem/discussions/160

**Comment by desputinski** [comment]
Hello/Welcome to the project!

**Comment by Zohreh-sudo** [comment]
Hi, 
I've got a message from one of you for contribution last week. How shall we communicate in this platform?

**Reply by Zohreh-sudo** [reply]
Alright now I read the info at the top.

**Reply by desputinski** [reply]
we also have discussion/chat groups in the privacy chat app "session":  https://getsession.org/ We can add you there, if you want to!

**Reply by armantorkzaban** [reply]
Hi @Zohreh-sudo, 
nice to have you here. Where would you like to begin? Would you like one of us to to show you around?

**Reply by Zedi-Bi** [reply]
Hi, 
Thanks for answering. Sorry for my late reply. I was terribly sick for 2 weeks. Where shall I start? I'll leave an intro in discussion.

**Reply by armantorkzaban** [reply]
Hi @Zedi-Bi, I have answered your message over Session. Still waiting for your reply.

**Comment by Zedi-Bi** [comment]
Hi, I'm Zedi and new here. My skills are in design thinking, system thinking and strategy. I'll be very happy to contribute in your project. It would be great if you could guide me through and let me know how I. Can contribute. 
Looking forward to hearing from you!
Best

**Reply by desputinski** [reply]
Welcome! Great, you want to join! With all questions concerning programming, it is probably best, if you phone or have a video call with Arman. With general questions we can also call/phone or chat. As said above, we have a group on the privacy messenger session. Best regards Christopher

**Reply by armantorkzaban** [reply]
Hi @Zedi-Bi, 
I am available over [Session](getsession.org) with this handle for a private talk: `051e78c4fa5aed198ab156a3be8ae039afe9fb3569e8c594856f15ebefd3054858`

In addition we could use this discussion board: https://github.com/tcfev/forDem/discussions/163

**Comment by fabifj** [comment]
Hi, I'm fabi and new here. My skills are in Design and Programming( JS Frameworks, Flutter). I'll be very happy to contribute in your project. 
You can assign me the tasks and I'll try to develop the front end of the App.

**Reply by armantorkzaban** [reply]
Hi @fabifj, 
this is wonderful. Welcome to the team. 
together with @zanzendegi we can assign fordem-app front-end tasks to you.
This will include making a Flutter version of Mastodon Android App. Are you ready?

**Comment by desputinski** [comment]
Hello fabi, that is great! We need people like you :) 
About the question which programming-tasks you can take part in, Arman will be able to brief you... 
Best regards

**Comment by zabuzara** [comment]
درود به همه دوستان،

من علاقه مندم به گروه توسعه نرمافزاری بپیوندم. تا بتونم کمکی کرده باشم.

Viele Grüße 
Omid
